test/__fixtures__/assemblies/@aws-cdk/aws-ecr@1.106.0/.jsii

```diff
@@ -238,6 +256,9 @@
       "namespace": "Amazon.CDK.AWS.ECR",
       "packageId": "Amazon.CDK.AWS.ECR"
     },
+    "go": {
+      "moduleName": "github.com/aws/aws-cdk-go"
```

Comment:
Just for clarity, should this be:

```suggestion
      "moduleName": "github.com/aws/aws-cdk-ecr-go"
```

Or something?

Reply:
no, the module name is the name of the repository where the package is located, which is correct. The package name is `awscdkawsecr` as generated from `jsii-pacmak` since it's not provided in the JSII go configuration making the full import path `github.com/aws/aws-cdk-go/awscdkawsecr` which is as expected.